### ISSUE TITLE
Function `map_meta_cap` was called incorrectly (WordPress 6.1)

### ISSUE THREAD

**Issue by Log1x** [short-pixel-optimizer/enable-media-replace#85]
When I go to edit a comment on one of my sites, it is throwing an exception due to a change in WordPress 6.1.

```php
Function map_meta_cap was called <strong>incorrectly</strong>. When checking for the <code>edit_post</code> capability, you must always check it against a specific post. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.1.0.)
```

It appears to be due to:

```php
if (current_user_can('edit_post', $post_id)  === true) {
```

found on line `317` in `classes/emr-plugin.php` where `$post_id` is returning `null`.

**Comment by pdobrescu** [comment]
Thanks for reporting this @Log1x, we'll look into it!

**Comment by pdobrescu** [comment]
dear @Log1x, can you please check with the latest version from the `updates` branch and let us know if it resolves the issue you have described? [92d9075](https://github.com/short-pixel-optimizer/enable-media-replace/commit/92d9075812c073a1a13f49e3a7c62c6c32f4de14) should basically fix this

**Comment by Log1x** [comment]
Sorry for the late reply– this did the trick. Thanks!
